op-test

```diff
@@ -94,12 +94,11 @@ import testcases
 args, remaining_args = OpTestConfiguration.conf.parse_args(sys.argv)
 OpTestConfiguration.conf.objs()
 
-print args
-
 class StandbySuite():
     '''Machine at standby. Focused on BMC'''
     def __init__(self):
         self.s = unittest.TestSuite()
+        OpTestLogger.optest_logger_glob.optest_logger.debug('addSuite of StandbySuite')
```

Comment:
Can we have global helper for getting logger at the top of the module and use that logger object in entire module instead of lengthy call backs.
```
logger = OpTestLogger.optest_logger_glob.optest_logger
logger.debug("msg")
```

Reply:
Done